command.go

```diff
@@ -258,8 +260,8 @@ Aliases:
 Examples:
 {{ .Example }}{{end}}{{ if .HasNonHelpSubCommands}}
 
-Available Commands: {{range .Commands}}{{if (not .IsHelpCommand)}}
-  {{rpad .Name .NamePadding }} {{.Short}}{{end}}{{end}}{{end}}{{ if .HasLocalFlags}}
+Available Commands: {{range .Commands}}{{if (not .IsHelpCommand)}}{{if (not .Hidden)}}
+  {{rpad .Name .NamePadding }} {{.Short}}{{end}}{{end}}{{end}}{{end}}{{ if .HasLocalFlags}}
```

Comment:
As it was, that was still necessary; however I just pushed some updates that has that removed from the template. To do so, I had to update/add some functions to better determine how commands should appear in the default usage/help text.